NuGetPackageVerifier.json

```diff
@@ -20,7 +21,12 @@
           "WRONG_PUBLICKEYTOKEN": {
             "tools/any/any/Newtonsoft.Json.dll": "Third party assembly bundled in our nupkg.",
             "tools/any/any/NuGet.Frameworks.dll": "Assembly built by another project but bundled in our nupkg.",
-            "tools/any/any/System.Runtime.CompilerServices.Unsafe.dll": "Assembly built by another project but bundled in our nupkg."
+            "tools/any/any/System.Buffers.dll": "Assembly built by another project but bundled in our nupkg.",
```

Comment:
FYI @davidfowl - this is a result of the change to Primitives

Reply:
Good to know, the span change?

Reply:
Why is it the result? Shouldn't it be trimmed?

Reply:
Should we cross compile primitives so we are not leaking package updates into 2.0 apps?

Reply:
Uh, I think no. Otherwise we'd end up doing that for a bunch of stuff no?

Reply:
Cross compile to what exactly? Doesn't it target `ns2.0` today?

Reply:
Cross-compile and reference 4.3.0 of system.memory in netstandard 2.0 version.

Reply:
OK, let's just fix the rest of validation issues then.

Reply:
@ryanbrandenburg could you follow up on the fix? You'll likely need to port the changes to this file to Coherence-Signed  too